invoices_test.go

```diff
@@ -363,6 +363,13 @@ func TestInvoices_Get(t *testing.T) {
                     <voidable type="boolean">true</voidable>
                     <refundable type="boolean">true</refundable>
                     <ip_address>127.0.0.1</ip_address>
+					<transaction_error>
```

Comment:
Minor nit but can you adjust the alignment to match up with the rest of the XML?